src/main/kotlin/com/openlattice/transporter/processors/TransporterSynchronizeTableDefinitionEntryProcessor.kt

```diff
@@ -0,0 +1,78 @@
+package com.openlattice.transporter.processors
+
+import com.hazelcast.core.Offloadable
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.assembler.AssemblerConnectionManager
+import com.openlattice.assembler.AssemblerConnectionManagerDependent
+import com.openlattice.edm.type.PropertyType
+import com.openlattice.transporter.tableDefinition
+import com.openlattice.transporter.types.TransporterColumnSet
+import org.slf4j.LoggerFactory
+import java.util.*
+
+data class TransporterSynchronizeTableDefinitionEntryProcessor(val newProperties: Collection<PropertyType>):
+        AbstractRhizomeEntryProcessor<UUID, TransporterColumnSet, Void?>(),
+        Offloadable,
+        AssemblerConnectionManagerDependent<TransporterSynchronizeTableDefinitionEntryProcessor> {
+    companion object {
+        private val logger = LoggerFactory.getLogger(TransporterSynchronizeTableDefinitionEntryProcessor::class.java)
+    }
+
+    private lateinit var acm: AssemblerConnectionManager
+
+    override fun process(entry: MutableMap.MutableEntry<UUID, TransporterColumnSet>): Void? {
```

Comment:
Any reason this and `transport()` can't be fused together?

Reply:
Honestly just to avoid having to keep jamming nulls into the early returns.